src/bindings/python/fluxacct/accounting/user_subcommands.py

```diff
@@ -280,6 +280,26 @@ def clear_queues(conn, username, bank=None):
     return 0
 
 
+def clear_projects(conn, username, bank=None):
+    update_stmt = "UPDATE association_table SET projects='*' WHERE username=?"
+    if bank is None:
+        conn.execute(update_stmt, (username,))
```

Comment:
SQL noob here but stray comma?

Reply:
Actually, `Connection` objects take a tuple for the parameter substitution when it is executing a `SQL` statement: 

> Instead, use the DB-API’s parameter substitution. To insert a variable into a query string, use a placeholder in the string, and substitute the actual values into the query by providing them as a [tuple](https://docs.python.org/3/library/stdtypes.html#tuple) of values to the second argument of the cursor’s [execute()](https://docs.python.org/3/library/sqlite3.html#sqlite3.Cursor.execute) method.

reference: https://docs.python.org/3/library/sqlite3.html